app/models/claim/advocate_claim.rb

```diff
@@ -228,6 +229,10 @@ def update_claim_document_owners
       documents.each { |d| d.update_column(:external_user_id, external_user_id) }
     end
 
+    def discontinuance?
+      case_type.name == 'Discontinuance'
```

Comment:
I'd make this based on the case_type.fee_type_code rather than its name/description as is less likely to change:

```
case_type.fee_type_code.eql?('GRDIS')
```

Reply:
👍